src/components/side_panel/chart/chart_with_axis/design_panel.xml

```diff
@@ -6,6 +6,14 @@
         <Section class="'pt-0'" title.translate="Values">
           <ChartShowValues t-props="props"/>
         </Section>
+        <Section class="'pt-1'" title.translate="Number formatting">
+          <Checkbox
+            name="'humanizeNumbers'"
+            label.translate="Humanize numbers"
+            value="props.definition.humanize ?? false"
+            onChange.bind="updateHumanizeNumbers"
+          />
+        </Section>
```

Comment:
It would be a good candidate for being a building block (like `ChartTitle` for exemple)